src/api/store/community.py

```diff
@@ -853,13 +853,23 @@ def can_use_this_subdomain(subdomain: str, community: Community=None) -> bool:
     return False
 
 def reserve_subdomain(subdomain: str, community: Community=None):
+    # first check that we can use this domain 
+    if not can_use_this_subdomain(subdomain, community):
+        return False
+
+    # if we are here then the subdomain is available to be used by this community
+    # now let's make sure it is all lower case
     subdomain = subdomain.lower()
 
-    if subdomain in RESERVED_SUBDOMAIN_LIST:
-        return False
-    # mark the old subdomains for this community to unused
+    # mark the old subdomains for this community to un-used
```

Comment:
Problem with this (if you don't return earlier) is you mark the Subdomain the community is using as 'not used'

Reply:
we set it on line 875